<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MessageForm</class>
 <widget class="QWidget" name="MessageForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Messenger</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/icons/16x16/message.png</normaloff>:/icons/16x16/message.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabBar" name="tabBar" native="true"/>
   </item>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <widget class="QTextEdit" name="dialogField">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="TypeField" name="typeField">
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TypeField</class>
   <extends>QTextEdit</extends>
   <header>type_field.h</header>
  </customwidget>
  <customwidget>
   <class>QTabBar</class>
   <extends>QWidget</extends>
   <header location="global">QTabBar</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
